common/dynamicconfig/constants.go

```diff
@@ -202,6 +202,8 @@ const (
 	FrontendThrottledLogRPS = "frontend.throttledLogRPS"
 	// FrontendShutdownDrainDuration is the duration of traffic drain during shutdown
 	FrontendShutdownDrainDuration = "frontend.shutdownDrainDuration"
+	// FrontendMembershipFailureDetectionDuration is the duration of membership failure detection
+	FrontendMembershipFailureDetectionDuration = "frontend.membershipFailureDetectionDuration"
```

Comment:
This is about grpc health checks (as done by an external load balancer or similar component), not membership. I don't think ringpop uses grpc health checks, does it?
```suggestion
	FrontendShutdownFailHealthcheckDuration = "frontend.membershipShutdownFailHealthcheckDuration"
```

Reply:
No. it doesn't

Reply:
although, maybe it also makes sense to add a call to `membershipMonitor.EvictSelf()` at the same time we start failing health checks? that will make workers stop sending rpcs to this frontend, I think